src/test/java/io/cdap/plugin/sfmc/source/MarketingCloudClientTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.sfmc.source;
+
+import com.custom.fuelsdk.PaginationETSoapObject;
+import com.exacttarget.fuelsdk.ETClient;
+import com.exacttarget.fuelsdk.ETExpression;
+import com.exacttarget.fuelsdk.ETSdkException;
+import io.cdap.plugin.sfmc.source.util.SourceObject;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ETExpression.class, ETClient.class, PaginationETSoapObject.class})
+public class MarketingCloudClientTest {
+
+  @Test
+  public void testParseExpressionFromValidateFilter() throws ETSdkException {
+    String filter = "";
+    PowerMockito.mockStatic(ETExpression.class);
+    ETExpression expression = PowerMockito.mock(ETExpression.class);
+    PowerMockito.when(ETExpression.parse("")).thenReturn(expression);
+    MarketingCloudClient.validateFilter(filter);
+    Assert.assertEquals(expression, ETExpression.parse(""));
```

Comment:
Isn't this going to be true because of line 40? What's the purpose of this test?

Reply:
Yes, so removed it.